mariner_proj/templates/html_export/e14ac900-ee59-11ed-94d5-0242ac120006.htm

```diff
@@ -0,0 +1,796 @@
+{% load template_tags %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wreck</title>
+    {% include 'html_export/milligram.htm' %}
+    {% include 'html_export/custom_style.htm' %}
+</head>
+<body>
+    <header>
+        {% include 'html_export/custom_header.htm' %}
+        <h1>Wreck</h1>
```

Comment:
The model name is Wreck Site rather than Wreck.  Should this be Wreck Site for consistency's sake or has the business requested this term?